### ISSUE TITLE
Differred payment for the same invoice

### ISSUE THREAD

**Issue by hacheni** [a-sit-plus/at-registrierkassen-mustercode#624]
Hello,

Sorry to write in English cause I do not speak German and I should implement the RKSV specification for our customer.

The system of our customer allows to make partial payments for a single invoice. For example, an invoice with 200€ can be paid in two stages : The first one the same day (lets say 100€) and the second one the next day for the remaining amount (100€).

For the two payments the system prints a bon of the same  invoice and the only difference is the marked payments. So the first bon that is printed will contain the payment of the first 100€ and mentions that it remains 100€ not paid. The second bon will contain the two payments. Both the two printed bons contains the total amount of the invoice which is 200€.

Behind the system creates two different business transactions with different identifiers. The first one is an Anzahlung and the second one is a Teilzahlung.

My questions is how to handle this case in term of RKSV specification ? Should I create a receipt having the "Anzahlung" business transaction ID as a number with an amount of 100€ and add the corresponding QRCode to the first printed bon then doing the same thing for the Teilzahlung business transaction when the second payment is done ? This means that finally I will have tow different bons of the same invoice : 
    * The first bon contains The ID of the Anzahlung as RKSV number and a QRCode showing the signature of the first payment of 100€
    * The second bon (it is the same invoice)  contains The ID of the Teilzahlung as RKSV number and a QRCode showing the signature of the second payment of 100€

Again sorry for writing in English.

Best regards

**Comment by ErichFreitag** [comment]
There are three ways to handle this - the one i hope easiest way in a short description:

a) you must take care that the invoice and the cash payment(s) do not create a duplicate tax dept. Easiest way to handle this is to mark the payment receipts as e.g. "duplicate - payment for invoice xy".

b) each cash payment is a separate RKSV transaction. In this case the amounts for paying an invoice could be put into 0 % tax rate (reason: tax is already a debt from the invoice itself).

So there is one transaction/receipt for the first 100 EUR, another transaction/receipt for the second 100 EUR (if both payed cash). As a consequence both receipts have different receipt numbers etc. as in any other cash transactions. Always the same - only difference is 0 % tax rate and each time referring to the invoice being payed.

If you want to add "Anzahlung" and "Teilzahlung" and "Restzahlung" on the receipts this will help for clearer understanding what happened but it is not necassary.

**Comment by hacheni** [comment]
Thank you for replying quickly,

Since we create a signature for each "Geschäftsfall" with the corresponding amount, I thing that solution (b) is suitable for our case.

**But** you marked (if both payed cash). What happen if the first payment is paid cach and the second one is paid by credit card ?

Another point we consider that all articles handled by our customer are 20% taxed (excepts Gutscheine). Why did you mention that the amounts could be put on the 0% tax rate ?

My last question if about "gutschrift" and it has no relation with what I asked before. The system we use does not allow to give back the money in case of  "gutschrift". But it gives a kind of a "Gutschein" with the equivalent amount. Should it be considred as a "storno" or a "null beleg" ?

Thanks for all

Best regards

**Comment by ErichFreitag** [comment]
Paying with credit card is also treated as cash payment - same as paying with EC and others.

You have to do both - a) and b) not either-or.

0 %: an invoice with tax (at GmbHs etc. in austria) leads to tax debt. A cash payment normally also leads to tax debt. When an invoice is payd in cash (= credit card, real-cash, EC, ...) this could make an assumption to pay tax twice. To avoid this the cash amount for paying the invoice should be noted in the 0 % tax rate (b) and a hint on the cash receipt shall be given that this payment is made for the invoice (a).

"Gutschrift" itself is not a cash transaction. It is only a paper where an amount is noted to be resepected at the next payment -therefore it is itself not RKSV relevant. It reduces the amount to pay in the next transaction. (it is allowed to handle this transaction also in 0 % tax rate, but it is not a requirement).

"Gutschrift" is comparable to a Wert-Gutschein (voucher only with amount, not bound to special kinds of articles, e.g. a theatre ticket). When you sell a Wert-Gutschein it is the same - selling the Amount-Voucher is only a change of payment method, not a cash transaction (except it is sold together with other articles - then it has to be noted in the 0 % tax rate, otherwise the turnover counter would not correctly reflect the payment value). Paying with the Amount-Voucher then is the same as a cash transaction and payment must be noted in the relevant tax rates as with "real" cash payment.

"Null-Beleg" is a receipt with a value 0, so this are normally only special transactions (Start-, Month-, Year-receipts and others). "Storno" is a special cancellation of an transaction (with influence to the turnover counter as a previous cash payment is cancelled)- "Gutschrift" is another business case (no cash payment is made).

**Comment by hacheni** [comment]
Thank you for all your clarifications.

Best resgards

**Comment by hacheni** [comment]
Hello,

I have some few questions about auszahlung (storno) and einzahlung (storno). Should they affect the turnover of the cashbox ? Should they be recorded on the 0% tax ?

Another question, what happens if a business transaction is recorded in DEP with 0€ for all amount categories and there are no receipt printed for the transaction. Is there any problem regarding to the law ? I ask this question because we have a "nachdruck" operation, used to reprint an old receipt and this lead to a specific business transaction. But it does not trigger any bon print for the operation itself. Since it was simpler for us to record in the DEP all business transaction, I wanted to know if recording such things in the DEP will make any contradictions with the law.

Best regards

**Comment by ErichFreitag** [comment]
Einzahlung and Auszahlung are no cash transactions. It is allowed to record these transactions in conjunction with RKSV, but then it has to be done in 0% tax rate. When you do it this also changes the turnover counter (always the same procedure). When you want to cancel such transactions with Storno this must be equivalent = also in 0 % tax rate.

If a business transaction results in 0,00 € cash value it is a normal transaction as any other. Therefore also a receipt must be printed. Example could be buying some articles and pay with a (value) voucher in the same height. The reason is simply to document the transaction, regardless of the amount to pay. Also always the same...

A Nachdruck is a copy of a previous transaction. It must be exactly the same receipt as the original receipt but
- without a change of the turnover counter
- clearly marked as "duplicate"
- therefore no DEP entry for it cause there is no cash transaction and no change of the turnover counter

So a duplicate of a receipt is no business transaction to be recorded (except in the cash-box journal, but not for RKSV).

**Comment by hacheni** [comment]
Thank you for replying,

It is clear for Einzahlung and Auszahlung but the answer for the second question was not really clear for me. Maybe because I did not explained well. But the main question was : 

is normal that a transaction recorded in DEP with 0 amount and does not have any proof receipt because we don't print it. The case of nachdruck was only to explain that the result of such transaction is the reprint of an old receipt without touching the turnover and the DEP because we take what was saved before. But the nachdruck it self is handled as a business transaction and so we record it in the DEP with 0 amount. The reason of this is that we decided to request for an RKSV signature of any  business transaction. However we don't  print any receipt for the nackdruck it self.

To summarize, Any entry in the DEP **must** have a proof receipt or not even for dummy operation ?

Best regards

**Comment by ErichFreitag** [comment]
Whatever your decision is or was: The case for Nachdruck (duplicate of a previous receipt) is specified as described above. It has NO DEP entry, no turnover-counter-change, NO cash value (also not 0) and it produces a receipt as described above - exactly the same receipt as the original one plus indication "Nachdruck"/"Duplikat". 

This is also NOT a dummy operation - it is no RKSV cash transaction, but a copy of a previous one. If you would make a DEP entry for this the duplicate receipt would get a new receipt number - but then it is not a copy. Also the receipt would normally show an amount different to 0, the DEP entry would be 0. This is not consistent.

Summary: any RKSV transaction you make leads to a receipt and to a DEP entry, regardless of the sum. This includes also all optional RKSV transactions. An RKSV implementation cannot avoid a minimum of business and decision logic to handle such and similar cases.

**Comment by hacheni** [comment]
Thank you for all your clarifications
